.github/workflows/mirrorchyan.yml

```
@@ -1,10 +1,11 @@
-name: mirrorchyan
+name: Release Package Distribution
```

```suggestion
name: MirrorChyan Release Package Distribution
```

I'd keep MirrorChyan in front since this will overwrite the .yml file name on the Actions tab making this harder to understand what it really is (since MirrorChyan is an external part of the project)

---

I thought that `Release Package Distribution` is more "open". Imagine we start to use more package distribution platforms in the future, and we won't need to change the workflow name, just add another job that says `Upload to MirrorChyanNo.2`.

---

The workflow is specifically named mirrorchyan and it uses MirrorChyan repo actions.